### DISCUSSION TITLE
Do you consider turning professional SD drawing pages into HTML+JavaScript?

### DISCUSSION THREAD

**Discussion by windkwbs** [LostRuins/lite.koboldai.net#76] [General]
Do you consider turning professional SD drawing pages into HTML+JavaScript?

**Comment by LostRuins** [comment]
I don't understand the question. Are you asking about the Stable UI? You can access that at http://localhost:5001/sdui

**Reply by windkwbs** [reply]
http://localhost:5001/sdui This interface currently needs to be made into pure HTML and JavaScript similar to lite. koboldai.net in order to be integrated into the webui provided by lite. koboldai.net,

**Reply by LostRuins** [reply]
Yes, it is standalone, but it cannot be run without koboldcpp as it uses the a1111 endpoint.
